tests/extractors/test_bigquery_extractor.py

```diff
@@ -75,17 +75,8 @@ def test_extract(self, mock_client, mock_hook):
             execution_date=datetime.utcnow().replace(tzinfo=pytz.utc))
 
         bq_extractor = BigQueryExtractor(task)
-
-        steps_meta = bq_extractor.extract()
-
-        assert steps_meta[0].name == "TestBigQueryExtractorE2E.task_id"
-        assert steps_meta[0].inputs == []
-        assert steps_meta[0].outputs == []
-        assert steps_meta[0].context is not None
-        assert steps_meta[0].context['sql'] == task.sql
-        assert steps_meta[0].context['bigquery.sql.parsed.inputs'] \
-            == '["customers"]'
-        assert steps_meta[0].context['bigquery.sql.parsed.outputs'] == '[]'
+        steps_meta_extract = bq_extractor.extract()
```

Comment:
I would change this test to call extract_on_complete instead since this is where the logic has moved.